src/main/base/Ball.java

```diff
@@ -134,6 +135,10 @@ public void hit(float d, int player, boolean ifSmash) {
             if (player == 1) {
                 currentSpeedX *= -1;
             }
+            new Sound("smash");
+        }
+        if(ifSmash==false){
+            new Sound("hit");
         }
```

Comment:
Always new a `Sound` object is not a good idea, maybe u can provide method in `Sound` to decide play which sound by send parameter